emscripten.py

```diff
@@ -238,6 +240,10 @@ def is_int(x):
         if DEBUG: logging.debug('very limited syscalls (%s) so disabling full filesystem support' % ', '.join(map(str, syscalls)))
         settings['NO_FILESYSTEM'] = 1
 
+    if settings['CYBERDWARF']:
+      settings['DEFAULT_LIBRARY_FUNCS_TO_INCLUDE'].append("emdebug_Debugger")
```

Comment:
we already have `emscripten_debugger` - what is this method, and how is it different?